### PR TITLE
Move documentation to Github

### PR DESCRIPTION
**Pull request by bjpalmer**
Most of the documentation on the www.gridpack.org has been converted to markdown files and moved to the Github repository. The top level README.md file is now acting as the GridPACK home page. This closes issue #133.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,64 @@ algebraic systems. Additional modules support input and output as well
...
 <!---See the [GridPACK home page](https://www.gridpack.org) for more information.-->
+The GridPACK framework will simplify parallel power grid application development
+by eliminating many of the index transformations and calculations required to
+distribute data in parallel applications and also to move data from one
+processor to another as the calculation proceeds. Application writers can focus
+primarily on the physics of their problem without worrying about much of the
+baggage associated with writing applications in parallel. GridPACK supplies core
+functionalities that will simplify parallel code development. These include a
+generic framework for setting up power grid networks and applying whatever
+physical properties to the bus and branches that developers want, functionality
```
Comment (abhyshr) [inline README.md:24]: Can we revert to the previous rather simple introduction to GridPACK? It was for power grid researchers. This one sounds more computer science concentrated.

File: README.md [hunk 2]
```diff
@@ -10,13 +10,64 @@ algebraic systems. Additional modules support input and output as well
...
+- [Downloads](https://github.com/GridOPTICS/GridPACK/releases)
+- [FAQS](docs/markdown/FAQS.md)
+- [Tutorials](docs/markdown/TUTORIALS.md)
+- [Data Sets](docs/markdown/DATASETS.md)
+- [License](docs/markdown/LICENSE.md)
+<!--
+- [Building GridPACK from Scratch](docs/markdown/BUILD_GRIDPACK.md)
+-->
+
+## Submodule(s)
```
Comment (abhyshr) [inline README.md:60]: Submodules can be moved to installation

File: README.md (hunk 1 above)
Comment (bjpalmer) [reply README.md:24]: I commented out the extra text. We can move it somewhere else if a good spot can be found for it.

File: README.md (hunk 2 above)
Comment (bjpalmer) [reply README.md:60]: I moved this text to the start of the basic install document. This information is also available in a few other places.

**Comment by bjpalmer** [comment]
I think this merge is ready to go.

**Pull request merged**